### PR TITLE
feature/BI-12229-Implementing-Configs

### PR DESCRIPTION
**Pull request by OsvaldoMartini**
Updating port for account validator api
updating "start.sh" file
updating README.file

### CODE REVIEW

File: routes.yaml [hunk 1]
```diff
@@ -2,9 +2,6 @@ app_name: account-validator-api
 group: api
 weight: 999998
 routes:
-  1: ^/account-validator/healthcheck
-  2: ^/account-validator/upload
-  3: ^/account-validator/files
-  4: ^/account-validator/files/{filename:.+}
-
+  1: ^/actuator/health
+  2: ^/account-validator/validade
```
Comment (modhasanjay) [inline routes.yaml:6]: Could you please correct the typo for validate?

Comment (OsvaldoMartini) [reply routes.yaml:6]: sure

Comment (OsvaldoMartini) [reply routes.yaml]: done

**Review by sbasile-ch — APPROVED** [review]
ok

**Pull request merged**